modules/node-observability-install-web-console.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/understanding-node-observability-operator.adoc
+
+:_content-type: PROCEDURE
+[id="install-node-observability-using-web-console_{context}"]
+= Installing the Node Observability Operator using the web console
+
+You can install the Node Observability Operator from the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+* You have access to the {product-title} web console.
+
+.Procedure
+
+. Log in to the {product-title} web console.
+. In the Administrator's navigation panel, expand *Operators* → *OperatorHub*.
+. In the *All items* field, enter *Node Observability Operator* and select the Node Observability Operator tile.
+. Click *Install*.
+. On the *Install Operator* page:
+.. *Update channel*: Select *alpha*.
```

Comment:
I cannot speak to the UI element that is required for these nested steps, but based on the IBM Style Guide: quick reference [1], specify from which UI element the user must take action. For example, `From the Update channel list, select alpha.`

[1] https://docs.engineering.redhat.com/display/CCSENABLE/UI+elements

Reply:
done.